c++/src/kj/refcount-test.c++

```diff
@@ -57,6 +57,55 @@ TEST(Refcount, Basic) {
 #endif
 }
 
+KJ_TEST("Rc") {
+  bool b = false;
+
+  Rc<SetTrueInDestructor> ref1 = kj::rc<SetTrueInDestructor>(&b);
```

Comment:
Not blocking for now but it might be good to have a test that mixes use of refcounted `kj::Own<T>` and `kj::Rc<T>` to make sure the behavior is as expected.

Reply:
since we don't expose `Own` inheritance, there is no mixed use of `kj::Own<T>` and `kj::Rc<T>` possible.